units/systemd-hibernate-resume-clear-efi.service.in

```diff
@@ -0,0 +1,24 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Clear Stale Hibernate EFI Variable
+Documentation=man:systemd-hibernate-resume-clear-efi.service(8)
+
+ConditionPathExists=/sys/firmware/efi/efivars/HibernateLocation-8cf2644b-4b0b-428f-9387-6d876050dc67
+ConditionPathExists=!/etc/initrd-release
+
+DefaultDependencies=no
+Before=sysinit.target shutdown.target
+Conflicts=shutdown.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
```

Comment:
Why `RemainAfterExit=yes`??

Reply:
Hmm, we have this on things like `systemd-boot-update.service` too. Given that this should really only run once during the whole boot, I think it's kinda reasonable?

Reply:
I see. Yeah, you are right.